ginga/web/pgw/ipg.py

```diff
@@ -535,9 +531,6 @@ def main(options, args):
     argprs.add_argument("--stderr", dest="logstderr", default=False,
                         action="store_true",
                         help="Copy logging also to stderr")
-    argprs.add_argument("--opencv", dest="use_opencv", default=False,
```

Comment:
Do you want to also issue a warning here instead of outright removing it? Will user call this via CLI directly?